<?xml version="1.0"?>

<launch>

  <node name="mocap_retime" pkg="mocap_retime" type="mocap_retime_node"  output="screen"> 
    <param name="simFlag" value="true" />
    <param name="pubName" value="/sim/ground/pose" />
    <param name="cutoffPosFreq" value="10" />
    <param name="cutoffVelFreq" value="10" />
  </node>
</launch>